drainer/schema.go

```diff
@@ -262,6 +262,17 @@ func (s *Schema) handlePreviousDDLJobIfNeed(version int64) error {
 	return nil
 }
 
+func skipFlash(job *model.Job) bool {
+	switch job.Type {
+	case model.ActionUpdateTiFlashReplicaStatus: // empty job.Query
+		return true
+	case model.ActionSetTiFlashReplica:
```

Comment:
No need to skip `ActionSetTiFlashReplica ` DDL job? this DDL has `Query`, How about synchronize this DDL and ignore the error?